data-prepper-plugins/anomaly-detector-processor/src/test/java/org/opensearch/dataprepper/plugins/processor/anomalydetector/AnomalyDetectorProcessorTests.java

```diff
@@ -146,19 +141,7 @@ void testAnomalyDetectorProcessor(int type) {
 
     @Test
     void testAnomalyDetectorProcessorTwoKeys() {
-        List<String> keyList = new ArrayList<String>();
-        keyList.add("latency");
```

Comment:
It is not clear to me why these test cases were touched. It is possible to keep the previous behavior, right? If not, it would be a breaking change.

Reply:
I was cleaning up by moving common setup to a shared @BeforeEach setup() method.